go_modules/helpers/updater/helpers.go

```diff
@@ -1,65 +0,0 @@
-package updater
-
-import (
-	"strings"
-
-	"golang.org/x/mod/modfile"
-)
-
-// Private methods lifted from the `modfile` package.
-// Last synced: 4/28/2021 from:
-// https://github.com/golang/mod/blob/858fdbee9c245c8109c359106e89c6b8d321f19c/modfile/rule.go
```

Comment:
@thepwagner @jurre this nicely solves this morning's discussion here: https://github.com/dependabot/dependabot-core/pull/3580#discussion_r622096596